app/controllers/account/chrome_extension_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Account::ChromeExtensionController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @companies = current_user.companies.all
+    @page = Page.new(source_url: params[:source_url])
+  end
+
+  def create
+    @company = current_user.companies.find(params[:page][:company_id])
```

Comment:
what if company_id was not selected in form ?

Reply:
![select](https://user-images.githubusercontent.com/27546645/34573669-91cb20c0-f17d-11e7-9680-78082396496b.png)
I've put 'selected' option for company's select, so that can be only one case when user cannot select company - if user hasn't any company
So, what I should to do in this case?

Reply:
render erros, that company must be present. just add validation for company_id presence: true